src/modules/launcher/PowerLauncher.UI/ResultList.xaml

```diff
@@ -1,101 +1,101 @@
-﻿<UserControl
-    x:Class="PowerLauncher.UI.ResultList"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d"
-    xmlns:ToolkitBehaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors"
-    xmlns:Core="using:Microsoft.Xaml.Interactions.Core"
-    xmlns:Interactivity="using:Microsoft.Xaml.Interactivity"
-    d:DesignHeight="300"
-    d:DesignWidth="720">
-    <Grid 
-        x:Name="PowerBar" 
-        Background="{StaticResource BackdropAcrylicBrush}"        

```

Comment:
Seems like only this line has been changed. Can we clean up the diff view to keep commit history clean?

Reply:
This seems to be some issue with git diff. I literally copy pasted the file `dev/powerLauncherCustomUI` and modified one line but it still shows it as changed.